examples/chat/RAG/recipes/utils.py

```diff
@@ -0,0 +1,106 @@
+import aiohttp
```

Comment:
This package doesn't seem to be installed after doing `pip install -e ".[dev]"`, so I think it needs to be added to `setup.cfg` explicitly.